tests/NHapi.NUnit/Parser/EncodingCharactersTests.cs

```diff
@@ -25,6 +25,35 @@ public void Constructor_ValidInput_ReturnsExpectedResult(char fieldSeperator, st
             Assert.AreEqual(encodingCharacters[3], sut.SubcomponentSeparator);
         }
 
+        [TestCase(null)]
+        [TestCase("")]
```

Comment:
test for null?

Reply:
not sure what you mean here?

Reply:
sorry, I meant test for whitespace if you make the other change